npk/src/npk.rs

```diff
@@ -692,7 +707,7 @@ fn sign_hashes(key_pair: &Keypair, hashes_yaml: &str) -> String {
     signature_yaml
 }
 
-fn copy_src_root_to_tmp(src: &Path, tmp: &TempDir) -> Result<PathBuf, Error> {
+async fn copy_src_root_to_tmp(src: &Path, tmp: &TempDir) -> Result<PathBuf, Error> {
```

Comment:
Just putting the `async` keyword before a rust function does not make the function asynchronous.
I this case it this function still blocks unless the call to `fs_extra::dir::copy` is dealt with correctly.
The same logic applies to all the other functions where just the `async` keyword was added.